README.md

```diff
@@ -1,13 +1,51 @@
-### NG Bootstrap - [Angular 2](http://angular.io/) directives specific to [Bootstrap](http://getbootstrap.com)
+# NG Bootstrap - [Angular 2](http://angular.io/) directives specific to [Bootstrap](http://getbootstrap.com)
 
 [![Build Status](https://travis-ci.org/ng-bootstrap/core.svg?branch=master)](https://travis-ci.org/ng-bootstrap/core)
 
-This library is a work in progress. Please check out our list of [issues](https://github.com/ng-bootstrap/core/issues) to see all the things we are implementing. Feel free to make any comment there ;)
+Welcome to the Angular 2 version of the [Angular UI Bootstrap](https://github.com/angular-ui/bootstrap) library.
+This library is being built from scratch in TypeScript using the Bootstrap 4 CSS framework.
 
-As for demos, we have a list of Plunkers with our components:
+As with Bootstrap 4 and Angular 2, this library is a work in progress. Please check out our list of
+[issues](https://github.com/ng-bootstrap/core/issues) to see all the things we are implementing.
+Feel free to make comments there.
+
+This project is being created by the [ui-bootstrap team](https://github.com/angular-ui/bootstrap).
+
+# Quick links
+- [Demo](#demo)
+- [Support](#support)
+    - [Code of Conduct](#code-of-conduct)
+    - [Supported browsers](#supported-browsers)
+- [Contributing to the project](#contributing-to-the-project)
+- [Development, meeting minutes, roadmap and more.](#development-meeting-minutes-roadmap-and-more)
+
+# Demo
 
 * [Accordion](http://plnkr.co/edit/XuH7mXBycODO7KYgNSbH)
 * [Dropdown](http://plnkr.co/edit/8FNKrEURwSzqqtsK5Rxn?p=preview)
 * [Rating](http://plnkr.co/edit/2wtcZ5bmQk8xmvwWvFP6?p=preview)
 
-This project is being created by the [ui-bootstrap team](https://github.com/angular-ui/bootstrap).
+# Support
+
+## Code of Conduct
+
+Please take a moment and read our [Code of Conduct](CODE_OF_CONDUCT.md)
+
+## Supported browsers
+
+We support the same browsers and versions supported by both Bootstrap 4 and Angular 2, which ever is _more_ restrictive.
+See [this](https://github.com/angular/angular/blob/master/README.md) for up-to-date Angular 2 browser support.
+
+* Chrome (45+)
+* Firefox (40+)
+* IE (9+) (but see [Bootstrap 4's notes](http://v4-alpha.getbootstrap.com/getting-started/browsers-devices/#internet-explorer-9) on IE9)
+* Edge (20+)
+* Safari (7+)
+
+## Contributing to the project
+
+We are always looking for the quality contributions! Please check the [CONTRIBUTING.md](CONTRIBUTING.md) doc for contribution guidelines.
+
+## Development, meeting minutes, roadmap and more.
```

Comment:
Sorry, the comments in the diff aren't clear.  Which line/section are you referring to?